config/software/dep-selector-libgecode.rb

```diff
@@ -36,8 +38,11 @@
 
   # Ruby DevKit ships with BSD Tar
   env["PROG_TAR"] = "bsdtar" if windows?
+  env["ARFLAGS"] = "rv #{env["ARFLAGS"]}" if env["ARFLAGS"]
```

Comment:
I was reading that if ARFLAGS isn't set, the default value is `rv` so it seems kinda weird that we tack on the `rv` here.